src/disco/utils.js

```diff
@@ -0,0 +1,7 @@
+import { sanitizeHTML } from 'core/utils';
+import purify from 'disco/purify';
+
+
+export const sanitizeHTMLWithNewTabLinks = (text, allowTags = []) => {
```

Comment:
I think `sanitizeHTMLWithExternalLinks` reads a bit better since opening target-blank URLs in a new tab can be changed in one's browser settings.